src/app/proficiency/controller.js

```diff
@@ -15,7 +15,7 @@ const completeTopic = async (req, res) => {
       })
     : await Proficiency.create({
         userId: userId.trim(),
-        topicId: topicId.trim(),
+        topicId: topic.dataValues.id,
```

Comment:
In your commit message, could you make this change more specific than `edit completeTopic function`?

Reply:
👍